web/src/app/admin/bot/SlackBotConfigCreationForm.tsx

```diff
@@ -247,6 +250,12 @@ export const SlackBotCreationForm = ({
                   label="Responds to Bot messages"
                   subtext="If not set, DanswerBot will always ignore messages from Bots"
                 />
+                <BooleanFormField
+                  name="enable_auto_filters"
+                  label="Enable LLM Autofiltering"
+                  subtext="If set, the LLM will attempt to filter out irrelevent documents based on 
+                            that docuement's source or when the docuement was last edited"
```

Comment:
remove typos. Also prefer something more positive/magical like:

If set, the LLM will generate source and time filters based on the user's query